config/shapes/process_selection.py

```diff
@@ -44,7 +153,25 @@ def prefiring_weight(era):
     return weight
 
 
-def MC_base_process_selection(channel, era):
+def MC_base_process_selection(channel, era, vs_jet_wp, vs_ele_wp):
+    wps_dict = {
+
+        "vvtight" : "VVTight",
+        "vtight" : "VVTight",
+        "tight" : "Tight",
+        "medium" : "Medium",
+        "loose" : "Loose",
+        "vloose" : "VLoose",
+        "vvloose" : "VVLoose",
+        "vvvloose" : "VVVLoose",
```

Comment:
why is this needed? Or crown naming are the ones with capital letters ? Why not use the same spelling for the shapes ?